### ISSUE TITLE
Nethash incorrect

### ISSUE THREAD

**Issue by Infernoman** [iquidus/explorer#9]
Hi there i got everything up and running im currently syncing the blocks into the database now. i noticed that the nethash is incorrect, currently thats because the coin im using is displayed in hashes / s  rather than gh/s i've got a bit of experience coding but am not familiar with what was used. so i was looking for a few pointers to get on the right track.
http://infernopool.com:8080/

**Comment by Infernoman** [comment]
By the way i absolutely love the system you have setup. with no knowledge of mongo interactions. and little knowledge of coding i was able to set this up in about 30 minutes. With very little hassle.

**Comment by iquidus** [comment]
Hi,

Thanks for submitting this issue (and the feedback), I may have to add an additional hashrate setting to handle this scenario as I have not encountered it before. It's a fairly strait forward fix, should be able to get to it tonight. In the mean time you could add a division to to return cb(body); line here:
https://github.com/iquidus/explorer/blob/master/lib/explorer.js#L37

e.g return cb(((body / 1024) / 1024) / 1024);

Let me know how you get on.

**Comment by Infernoman** [comment]
Perfect. absolutely love the ease of use to this explorer. job well done ! once im all synced up ill let you know if i find anything else.

**Comment by Infernoman** [comment]
Everything is working great now once again i applaud your work that youve done. if i may make a suggestion. possibly that you add the market cap for the coin onto the block explorer?

**Comment by iquidus** [comment]
Turns out it was a result of the refactoring that I did, removed the conversion to gh/s and did not add it back. Just pushed a fix to master branch. 6a2145cff75e50e0783183673ae1f3972952eedf

Marketcap is already planned for the next update. I really like the burn type suggestion. Just added your instance to the readme. 

Thanks for reporting this so quickly, the great feedback and your suggestions, all much appreciated.
